lstchain/datachecks/dl1_checker.py

```diff
@@ -1436,16 +1436,10 @@ def get_pixel_location(pix_id):
     # The first time we read in the data stored in the io directory:
     infilename = Path(Path(os.path.dirname(__file__)).parent,'io',
```

Comment:
Better use
`from pkg_resources import resource_filename` and `resource_filename('lstchain', 'resources/LST_pixid_to_cluster.txt').`

This is also where the file should go, otherwise you will need to adapt the setup py to also include this file in the distributed package explicitly.

Reply:
Ah, sorry. resources is not setup. So you have to include this file in package_data in the `setup.py` file

Reply:
Done, thanks!

Reply:
> Ah, sorry. resources is not setup. So you have to include this file in package_data in the `setup.py` file

I had not seen this. Done now.